x-pack/platform/plugins/shared/streams/server/lib/significant_events/prompts/kql_guide.text

```diff
@@ -1,51 +1,39 @@
-## KQL
+## KQL Query Syntax Guide
 
-KQL (Kibana Query Language) is a query language available in Kibana, that will be converted into Elasticsearch Query DSL.
+### Core Syntax Rule
+**`field:value`** - Search for `value` within the specified `field`
 
-- Supports auto completion of fields and values
-- Supports searching on scripted fields
-- Supports wildcard on field names
-- Simpler syntax for some operators
+### Key Principles
 
-### Finding values
-Find documents where any field matches any of the words/terms listed. The term must
-appear as it is in the document, e.g. this query won't match documents containing
-the word "darker": `dark light`.
+#### 1. Exact Phrase Matching
+- **Quotes preserve exact phrases**: `message:"dark cat"` finds documents where the `message` field contains the exact phrase "dark cat"
```

Comment:
I think wrapping in quotes means that they need to appear in that order, but they do not need to be the exact phrase. Take a look at https://www.timroes.de/kibana-search-cheatsheet.